tests/src/Unit/AnnotationTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
This file does not only have two unnecessary methods (`__construct` and `setUp`), but its name is not descriptive. Why is it called AnnotationTest? You're not testing anything like that. You're literally just creating an instance of `Network` and checking if a property is of type array (something that I consider as silly as the tests I mentioned last time). Please, think about what your tests are going to do before writing any code.

Reply:
The `setUp` method is just in case we need to call something in all the tests in a class, we can add it to this `setUp` method which will run before all the methods. But, I have removed them from the tests, we can add them later if required.

Reply:
What I meant is that if the whole purpose of `methodA` is to call `parent::methodA', then you don't need to declare `methodA`. Inheritance will take care of things.